calyx-py/test/arb_hw_6.futil

```diff
@@ -0,0 +1,373 @@
+import "primitives/core.futil";
+
+component wrap2(i: 32, j: 32) -> () {
```

Comment:
You might consider calling this `arb_6_to_12` or similar to convey its purpose a bit.